handler-bizkeeper/src/main/java/io/servicecomb/bizkeeper/BizkeeperHandler.java

```diff
@@ -52,6 +54,16 @@
         } catch (Exception e) {
             LOG.warn("Hystrix properties already registerd. Dynamic configuration may not work.");
         }
+        try {
+            HystrixPlugins.getInstance().registerCommandExecutionHook(new HystrixCommandExecutionHook() {
+                public <T> Exception onExecutionError(HystrixInvokable<T> commandInstance, Exception e) {
+                    LOG.warn("bizkeeper execution error", e);
+                    return e; //by default, just pass through
+                }
+            });
+        } catch (Exception e) {
```

Comment:
As you don't log the actual exception message, it could mislead the user if the RuntimeException is not related to HystrixCommandExcecution hook setting.